package.json

```diff
@@ -30,7 +30,8 @@
     "st": "^1.2.0",
     "touch": "^1.0.0",
     "whole-line-stream": "^0.1.0",
-    "yauzl": "^2.6.0"
+    "yauzl": "^2.6.0",
+    "iphone-inline-video": "^1.9.3"
```

Comment:
I've kept this list sorted alphabetically for easier maintainance. So please insert the new line appropriately. (This has the benefit of this PR just modifying one line, not two.)